heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/metricscache/MetricsCacheQueryUtils.java

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Twitter. All rights reserved.
```

Comment:
Sorry, I might have referenced the wrong classes, I meant the ones in this review that seem to mimick the protobuf objects, if that's in fact what they do.